libbpf-tools/Makefile

```diff
@@ -111,6 +123,22 @@ $(OUTPUT)/%.bpf.o: %.bpf.c $(LIBBPF_OBJ) $(wildcard %.h) $(ARCH)/vmlinux.h | $(O
 		     -I$(ARCH)/ $(INCLUDES) -c $(filter %.c,$^) -o $@ &&      \
 	$(LLVM_STRIP) -g $@
 
+ifdef ENABLE_MIN_CORE_BTFS
+
+.PHONY: update_btfhub
+update_btfhub: $(BTF_HUB_ARCHIVE)
+	$(Q)cd $(BTF_HUB_ARCHIVE) && git pull
+
+$(BTF_HUB_ARCHIVE):
+	$(call msg,GIT CLONE,$@)
+	$(Q)git clone https://github.com/aquasecurity/btfhub-archive/ $(BTF_HUB_ARCHIVE)
+
+BPF_O_FILES = $(patsubst %,$(OUTPUT)/%.bpf.o,$(APPS))
+
+$(OUTPUT)/min_core_btf_tar.o: $(BPF_O_FILES) update_btfhub
+	$(Q)$(MAKE) -f btfgen.mk
```

Comment:
naming nit: btfgen.mk -> Makefile.btfgen? It's much easier to spot makefile that actually spells out "Makefile" in the name